gyp/pylib/gyp/MSVSNew.py

```diff
@@ -59,10 +49,11 @@ def MakeGuid(name, seed='msvs_new'):
 #------------------------------------------------------------------------------
 
 
+@total_ordering
 class MSVSSolutionEntry(object):
-  def __cmp__(self, other):
+  def __lt__(self, other):
```

Comment:
Sorry if that's a dumb question, but should we also define `__eq__` and `__ne__` as per the page you linked to? 

> To avoid the hassle of providing all six functions, you can implement `__eq__`, `__ne__`, and only one of the ordering operators, and use the `functools.total_ordering()` decorator to fill in the rest.